blur_no-alt.php

```diff
@@ -54,70 +86,169 @@ public function blur_no_alt_message_display_create_admin_page() {
 				?>
 			</form>
 		</div>
-	<?php }
+		<?php
+	}
 
+	/**
+	 * Register Settings page, sections, and settings
+	 */
 	public function blur_no_alt_message_display_page_init() {
 		register_setting(
-			'blur_no_alt_message_display_option_group', // option_group
-			'blur_no_alt_message_display_option_name', // option_name
+			'blur_no_alt_message_display_option_group',
+			'blur_no_alt_message_display_option_name',
 			array(
+				'description'       => esc_html_x( 'All settings for the Blur No-Alt plugin', 'Description property of register_setting function', 'blur-no-alt' ),
+				'type'              => 'array',
 				'sanitize_callback' => array( $this, 'blur_no_alt_message_display_sanitize' ),
-				'default' => $this->blur_no_alt_message_display_options_default,
+				'default'           => $this->blur_no_alt_message_display_options_default,
 			)
 		);
 
 		add_settings_section(
-			'blur_no_alt_message_display_setting_section', // id
-			__( 'Editor Settings', 'blur-no-alt' ), // title
-			array( $this, 'blur_no_alt_message_display_section_info' ), // callback
-			'blur-no-alt-message-display-admin' // page
+			'blur_no_alt_message_display_setting_section',
+			esc_html__( 'Editor Settings', 'blur-no-alt' ),
+			false,
+			'blur-no-alt-message-display-admin'
+		);
+
+		add_settings_section(
+			'blur_no_alt_front_end_blur_setting_section',
+			esc_html__( 'Front-end Settings', 'blur-no-alt' ),
+			array( $this, 'blur_no_alt_front_end_section_callback' ),
+			'blur-no-alt-message-display-admin',
+		);
+
+		add_settings_field(
+			'show_or_hide_0',
+			esc_html__( 'Message above Block Editor title explaining blurred images', 'blur-no-alt' ),
+			array( $this, 'show_or_hide_setting_callback' ),
+			'blur-no-alt-message-display-admin',
+			'blur_no_alt_message_display_setting_section'
 		);
 
 		add_settings_field(
-			'show_or_hide_0', // id
-			__( 'Message explaining blurred images at top of editor page', 'blur-no-alt' ), // title
-			array( $this, 'show_or_hide_0_callback' ), // callback
-			'blur-no-alt-message-display-admin', // page
-			'blur_no_alt_message_display_setting_section' // section
```

Comment:
Does this effectively remove the option to show/hide the explanatory banner? Or is that elsewhere?

Reply:
Oh duh, I see it above. Can the comments go back, though?